### DISCUSSION TITLE
rules/config(other than realert time) change to bring down the elastalert counts to maximum of 20 alerts per day.

### DISCUSSION THREAD

**Discussion by Singh-Shivani** [jertel/elastalert2#645] [Q&A] [answered]
Hi,
Is there any other way to limit elastalert generating alert email only till a specific count(other than increasing re-alert time)?

**Comment by jertel** [comment, marked as answer]
[exponential_realert](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#exponential-realert) can be used as well, but otherwise I'm not aware of an absolute max alerts option.
